### ISSUE TITLE
UI: contact context menu too small

### ISSUE THREAD

**Issue by streaps** [cmeng-git/atalk-android#14]
A long touch on a contact opens the context menu:
```
Close current chat session
Close all active chats
[...]
Contact Info
Cancel
```
The menu is displayed above the touch point (your finger). On the first user the menu is very small (only one line in height), because there is not much space available above the touch point.

**Comment by cmeng-git** [comment]
The direction of the context expansion is not decided by aTalk, but by android operating system itself. I have once contacted android OS developer on this problem. Their last feedback mentioned that the problem is not due to android OS, but the firmware of the device manufacturer. I have the same problem on my Note 8, but not on Note 3. I try to give feedback to Samsung, but unable to find suitable forum to give my comment.

**Comment by streaps** [comment]
Thanks for the info. I'm using LineageOS 14.1 (Android 7.1) on a Samsung Galaxy S3. I will test it later on another Galaxy S3 with the Samsung Android 4.3 ROM.

**Comment by streaps** [comment]
On the Galaxy S3 with Android 4.3 everything looks fine, the menu is always the same size no matter at which position you tap.

**Comment by cmeng-git** [comment]
I have report the problem back to android team. For any update of the problem, please refer to
https://issuetracker.google.com/issues/111317442

**Comment by cmeng-git** [comment]
See latest update:
https://issuetracker.google.com/issues/113072045

**Comment by cmeng-git** [comment]
I decided to close this issue as the final solution to fix the problem lies with android OS development team.

**Comment by cmeng-git** [comment]
Problem fixed in aTalk v2.1.7; Implement the Context Menu in both the Contact iist and ChatRoom list in aTalk using PopUpMenu instead. Android development team is still unable to fix the android own ContextMenu when use in Fragment.
